Source/JavaScriptCore/assembler/MacroAssemblerARM64.h

```diff
@@ -5211,7 +5211,7 @@ class MacroAssemblerARM64 : public AbstractMacroAssembler<Assembler> {
 
     static void reemitInitialMoveWithPatch(void* address, void* value)
     {
-        Assembler::setPointer<jitMemcpyRepatchFlush>(static_cast<int*>(address), value, dataTempRegister);
+        Assembler::setPointer<jitMemcpyAtomicRepatchFlush>(static_cast<int*>(address), value, dataTempRegister);
```

Comment:
`setPointer` is not used by common code.  So, ARM64 an X86_64 port shouldn't need it to be parameterized.  Perhaps the parameter should be removed to simplify the code.